### ISSUE TITLE
py3 development version seems to submit DN as lower case, hence not recognising existing DN

### ISSUE THREAD

**Issue by marianne013** [sfayer/cert_sorcerer#8]
```deathstar:~ :~] ./CS.py30 /C=UK/O=eScience/OU=Imperial/L=Physics/CN=daniela bauer
Processing USER cert with "CN=/c=uk/o=escience/ou=imperial/l=physics/cn=daniela bauer" (OU=Imperial,L=Physics,O=eScience,C=UK).
There is no local data about this DN, request a new cert [y/N]? N
Abort ... abort... abort....
deathstar:~ :~] openssl x509 -in .globus/usercert.pem -noout -subject
subject=C = UK, O = eScience, OU = Imperial, L = Physics, CN = daniela bauer
```

**Comment by marianne013** [comment]
Never mind I read the instructions wrong, but now I get:
```
deathstar:~ :~] ./CS.py30 "daniela bauer"
Processing USER cert with "CN=daniela bauer" (OU=Imperial,L=Physics,O=eScience,C=UK).
There is no local data about this DN, request a new cert [y/N]? N
Cancelling...
```

**Comment by marianne013** [comment]
I should have sent you an email:
```
deathstar:~ :~] ./CS.py30 "daniela bauer"
Processing USER cert with "CN=daniela bauer" (OU=Imperial,L=Physics,O=eScience,C=UK).
This certificate has been previously signed, start a renewal [y/N]? y
Please enter a PIN for this renewal (min 10 chars): totallyfakepwd
Traceback (most recent call last):
  File "/home/dbauer/./CS.py30", line 839, in <module>
    CS_UI.renew_cert(store, cn, hostcert)
  File "/home/dbauer/./CS.py30", line 701, in renew_cert
    sha.update(userpin)
TypeError: Unicode-objects must be encoded before hashing
```
alternatively is there any chance you could install the missing packages on lx02, so I can run it there ?

**Comment by sfayer** [comment]
Fixed in commit 0e05d27466926bd956b2b5d0c8a06c8ba3e40eb2.
